.circleci/config.yml

```diff
@@ -339,6 +339,20 @@ commands:
           path: fastlane/test_output/xctest
           destination: scan-test-output
 
+  push-to-spm:
+    steps:
+      - checkout
+      - setup-git-credentials
+      - run:
+          name: Clone purchases-ios and push to purchases-ios-spm
+          command: |
+            git clone https://github.com/RevenueCat/purchases-ios.git
+            cd purchases-ios
+            git fetch --tags
+            git remote set-url origin https://github.com/RevenueCat/purchases-ios-spm.git
+            git push origin
+            git push --tags
```

Comment:
[not needed for this PR] ideally we'd move this to fastlane, we try to keep the CI stuff as simple as possible and move stuff into fastlane which is much easier to test. 
Also it feels like the URLs might be a good thing to move into env variables in case we rename them, but not a dealbreaker and we might never need it in any case.